src/main/java/org/purescript/psi/import/PSImportedItem.kt

```diff
@@ -0,0 +1,59 @@
+package org.purescript.psi.import
+
+import com.intellij.lang.ASTNode
+import org.purescript.psi.PSIdentifierImpl
+import org.purescript.psi.PSProperName
+import org.purescript.psi.PSPsiElement
+
+sealed class PSImportedItem(node: ASTNode) : PSPsiElement(node) {
+    abstract override fun getName(): String
+}
```

Comment:
This file is not really added, it's simply git being confused since I changed the package.